app/views/jobseekers/profiles/about_you/show.html.slim

```diff
@@ -7,10 +7,6 @@
   .govuk-grid-column-two-thirds
     h1.govuk-heading-l = t(".page_title")
 
-    = govuk_summary_list do |summary_list|
-      - summary_list.row do |row|
-        - row.key text: t(".summary_title")
-        - row.value text: @profile.about_you
-        - row.action(text: t("buttons.change"), href: edit_jobseekers_profile_about_you_path)
+    == render(partial: "summary", locals: { profile: @profile, title: t(".summary_title") })
```

Comment:
Why the need for a partial here?

Reply:
so it can be used on review page and the main profile index page